README.md

```diff
@@ -100,18 +100,18 @@ $ docker run -d -e MONGODB_USER=<user> -e MONGODB_PASSWORD=<password> -e MONGODB
 
 If you are initializing the database and it's the first time you are using the
 specified shared volume, the database will be created, together with database
-administrator user and also MongoDB admin user if `MONGODB_ADMIN_PASSWORD`
-environment variable is specified. After that the MongoDB daemon will be started.
-If you are re-attaching the volume to another container the creation of the
-database user and the admin user will be skipped and only the mongodb daemon
-will be started.
+administrator user and also MongoDB admin user. After that the MongoDB daemon 
+will be started. If you are re-attaching the volume to another container the 
+creation of the database user and the admin user will be skipped and only the 
+mongodb daemon will be started.
 
 
 MongoDB admin user
 ---------------------------------
-The admin user is not set by default. You can create one by setting
-`MONGODB_ADMIN_PASSWORD` environment variable, in which case the admin user name
-will be set to `admin`. This process is done upon initializing the database.
+
+The admin user is set by default. Admin user name is set to `admin` and you
+have to to specify his password by setting `MONGODB_ADMIN_PASSWORD` environment 
+variable. This process is done upon initializing the database.
```

Comment:
Can you please remove this entire section and just put one sentence somewhere else in the readme about the admin username being `admin`? I think the first sentence is now redundant and saying the user needs to set the admin password is also not necessary, since it is known from the list of env vars.